src/System.Net.Sockets/tests/FunctionalTests/AcceptAsync.cs

```diff
@@ -8,7 +11,7 @@ namespace System.Net.Sockets.Tests
 {
     public class AcceptAsync
     {
-        private const int TestPortBase = 8000;
+        private const int TestPortBase = 7000;
```

Comment:
Two reasons:
1. TestPortBases exists strictly to accommodate the fact that the same sources are built twice into two different assemblies that may run in parallel (and thus collide). These sources do not share this characteristic.
2. Most/all of this manual port management should disappear in the future by transitioning to a plan wherein the OS automatically assigns port numbers. Despite the changes I'm making here, this may need to happen sooner rather than later in order to allow two test runs to happen on the same machine. If I can't figure this out today, the only other option is to disable the Linux tests in the lab for the time being.